src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -657,13 +658,21 @@ public DarCollection cancelDarCollectionElectionsAsChair(DarCollection collectio
    * @param collection The DarCollection
    * @return The updated DarCollection
    */
-  public DarCollection createElectionsForDarCollection(User user, DarCollection collection) {
+  public DarCollection createElectionsForDarCollection(User user, DarCollection collection)
+      throws Exception {
     try {
       List<String> createdElectionReferenceIds = collectionServiceDAO.createElectionsForDarCollection(
           user, collection);
-      List<User> voteUsers = voteDAO.findVoteUsersByElectionReferenceIdList(
-          createdElectionReferenceIds);
+      if (createdElectionReferenceIds == null || createdElectionReferenceIds.isEmpty()) {
+        var e = new IllegalStateException(
+            "No elections were created for DAR Collection: %s".formatted(
+                collection.getDarCode()));
+        logException(e);
+        throw e;
+      }
       try {
```

Comment:
Nesting `try`s should be avoided when possible. In this case, both the inner and outer `try` catch `Exception`, so the code should work the same if this nested `try` of moved after the outer `catch` to line 677 below

Reply:
@rushtong suggested I scope `findVoteUsersByElectionReferenceIdList` to ignore email failures but not election creation errors. I otherwise didn't modify the logic here.

Reply:
Yes, my point is that the original code was nesting `try`s where it didn't need to do so.